contrib/localnet/README.md

```diff
@@ -45,6 +45,24 @@ To stop the localnet:
 docker-compose down
 ```
 
+### Running in chaos mode
```

Comment:
Could we move or copy these in this file https://github.com/zeta-chain/node/blob/0d5ac2ae3f1d3cfc409488dc1ec9268a3b080db2/docs/development/LOCAL_TESTING.md#L1